ifrs17/Import/Importers.ipynb

```diff
@@ -377,6 +375,34 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "code",
+            "source": [
+                "public async Task<ActivityLog> CalculateAndUploadIfrsVariablesAsync(ImportArgs args, IWorkspace workspace)",
```

Comment:
Create a sub section for this import helper named "Calculate and Upload IFRS Variables"

Reply:
done